src/EnergyPlus/Plant/Solution/PlantLoopSolver.cc

```diff
@@ -726,6 +719,11 @@ namespace PlantLoopSolver {
 				auto & branch( loop_side.Branch( BranchCounter ) );
 
 				if ( BranchCounter > 1 && BranchCounter < NumBranchesOnThisLoopSide ) ++ParallelBranchIndex;
+
+				if ( branch.QuickDisableForCSBranchPumping ) {
```

Comment:
Here we check to see if the branch was disabled for this fix, and if so, we don't allocate any flow request for that branch.  It has the effect of turning off the branch.